README.md

```diff
@@ -1,39 +1,108 @@
 # Amazon Textract Document Understanding Solution
 
+## CICD Deploy
+
+### Requirements
+
+- aws cli
+
+### Getting Started with Full Deploy
+
+- Create a bucket to act as the target Amazon S3 distribution bucket
+
+_Note:_ You will have to create an S3 bucket with the template 'my-bucket-name-<aws_region>'; aws_region is where you are testing the customized solution. Also, the assets in this bucket should be publicly accessible.
+
+For example, you create a bucket called `document-understanding-bucket-us-east-1`,
```

Comment:
NIT : Seems that now ther Solutions Builder team has moved to store all solutions in one bucket with each solution separated by prefix as shown athttps://github.com/awslabs/smart-product-solution/blame/master/README.md#L22
We could use another example name such as "my-solutions-bucket-us-east-1"